boms/bom/pom.xml

```diff
@@ -88,19 +88,10 @@
         <version>1.3</version>
         <type>pom</type>
       </dependency>
-      <!-- Java Persistence API (2.x not available in groupId javax.persistence) -->
       <dependency>
-        <groupId>org.hibernate.javax.persistence</groupId>
-        <artifactId>hibernate-jpa-2.1-api</artifactId>
-        <version>1.0.0.Final</version>
-      </dependency>
-      <!--
-        <dependency>
         <groupId>javax.persistence</groupId>
         <artifactId>javax.persistence-api</artifactId>
-        <version>2.2</version>
```

Comment:
why did you remove the version? IMHO this is a mistake. A dependency in depencenyManagement without a version seems pointless to me. Or is this a miss-interpretation of the diffs?
https://github.com/devonfw/devon4j/blob/d9d7403493513a24a14b363d3701fbd67b6d72ce/boms/bom/pom.xml#L93

Reply:
If that dependency comes in 2.2+ version via springs BOMs and we do not want to repeat it here, you should remove it entirely.

Reply:
Nice catch @hohwille , javax.persistence-api:2.2 is already defined in spring-boot-depenedencies:2.1.2.RELEASE. So I think it is better to remove
it completely from devon4j-bom. Removed and changes pushed to the PR.